Hush/Assets/Scripts/Game/GameMaster.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class GameMaster : MonoBehaviour
+{
+    public static GameMaster gm;
+
+    private static int numOfKeys = 0;
+    public static int NumOfKeys {
+        get {
+            return numOfKeys;
+        }
+    }
+    private static bool hasRelic = false;
+    public static bool HasRelic {
+        get {
+            return hasRelic;
+        }
+    }
+
+    void Awake()
+    {
+        DontDestroyOnLoad(this.gameObject);
+    }
+    
+    // Start is called before the first frame update
+    void Start()
+    {
+        if (gm == null) {
+			gm = GameObject.FindGameObjectWithTag ("GM").GetComponent<GameMaster>();
+		}
+    }
```

Comment:
Shouldn't need this with the changes proposed above

Reply:
Just fix the formatting here 😋

Reply:
![image](https://user-images.githubusercontent.com/49250036/140006526-e0277855-0f1f-4f20-9578-5140c1f6f68f.png)
I see it like this on my end?

Reply:
![image](https://user-images.githubusercontent.com/49250036/140006682-592ec5b7-b3f2-4856-b5bf-936787ac8662.png)
even here i dont see that weird formatting